### ISSUE TITLE
sms-spec dependency on rspec-mocks (~> 2.12.0) prevents upgrading app to use rspec 2.13.0

### ISSUE THREAD

**Issue by jpwynn** [mhs/sms-spec#7]
When trying to do 'bundle update rspec-rails' it looks like a dependency prevents the upgrade. I suspect the dependency could be safely updates, and perhasp should be 
~> 2.12 
(if tht would permit an update to 2.13.0) instead of 
~2.12.0
which I think means >= 2.12.0 but < 2.13

Bundler could not find compatible versions for gem "rspec-mocks":
  In Gemfile:
    sms-spec (>= 0) ruby depends on
      rspec-mocks (~> 2.12.0) ruby

```
rspec-rails (= 2.13.0) ruby depends on
  rspec-mocks (2.13.0)
```

**Comment by noreaster** [comment]
Yeah, I might have to drop this gem, because in order to use a modern version of rspec (3.2.0), I can't use the most recent version of this gem. (only version that would install was 1.7).  The rspec-mocks dependency was holding me back from doing updates.
